SCClassLibrary/Common/Audio/UGen.sc

```diff
@@ -576,8 +576,9 @@ MultiOutUGen : UGen {
 	}
 
 	initOutputs { arg numChannels, rate;
-		if(numChannels.isNil or: { numChannels < 1 }, {
-			Error("%: wrong number of channels (%)".format(this, numChannels)).throw
+		if(numChannels.isInteger.not or: { numChannels < 1 }, {
```

Comment:
One question: if `numChannels` turns out to be a float, we could also convert it to an Integer here, couldn't we?

Reply:
I'm not a fan of coercion/truncation of argument values. For instance if someone passes in `1.5` it seems better to simply fail than to soldier on.

Reply:
The general way of thinking in supercollider is to pass everything to the latest point of failure. And generally speaking there is an implicit understanding that all numbers are converted to their correct versions by the UGens themselves.

But in this case yes, it makes sense.